pkg/ipmi/ipmi.go

```diff
@@ -41,6 +44,9 @@ const (
 	_IPM_WATCHDOG_SMS_OS       = 0x04
 	_IPM_WATCHDOG_CLEAR_SMS_OS = 0x10
 
+	_IPMI_NETFN_STORAGE = 0xA
+	_BMC_ADD_SEL        = 0x44
```

Comment:
Maybe group this one with other ipmi commands such as `_BMC_GET_WATCHDOG_TIMER`.